### DISCUSSION TITLE
Rebase Commits Aren't Verified

### DISCUSSION THREAD

**Discussion by MarvNC** [community/community#114334] [Pull Requests]
### Select Topic Area

Bug

### Body

Usually I see that merges and such done via the web UI are verified. It seems that doing a rebase to merge changes from main/master using the web UI results in an unverified commit: https://github.com/themoeway/yomitan/pull/781

**Comment by devashish2024** [comment]
Hey there, @MarvNC 

This is not a issue, you can configure your account to **Flag Unsigned commits as unverified**.

1. In the upper-right corner of any page, click your profile photo, then click Settings.
2. In the "Access" section of the sidebar, click  SSH and GPG keys.
3. Under "Vigilant mode," select Flag unsigned commits as unverified.

That is it. Visit [here](https://docs.github.com/en/authentication/managing-commit-signature-verification/displaying-verification-statuses-for-all-of-your-commits) for the source!

**Reply by MarvNC** [reply]
This does not resolve this issue, I already have that feature enabled which is why I'm asking.
